### DISCUSSION TITLE
Can you run standalone version?

### DISCUSSION THREAD

**Discussion by jcubic** [GoogleChrome/lighthouse#15247] [Q&A]
Lighthouse doesn't work right now on Linux it hangs with the message "lighthouse is warming up". I cannot run it on Fedora and Ubuntu 114.0.5735.133 (official 64bit version).

So is there an alternative? Like running as a standalone app without dev tools?

**Comment by adamraine** [comment]
Yes! We have a few different clients, including a CLI: https://github.com/GoogleChrome/lighthouse#using-the-node-cli

#15026 should be resolved in M115
